### DISCUSSION TITLE
livewire-datatables with raw query

### DISCUSSION THREAD

**Discussion by sixtyrc** [livewire/livewire#4490] [1. Help]
Hello, it is possible to use a raw query instead of a model to use the livewire datables, all the examples that I see always use for example
<livewire: datatable model = "App \ User" exclude = "planet_id, bio, latitude, longitude, updated_at" />

instead of the model, I would like to use for example: $ pac = DB :: select ('EXEC IHByDate "'. $ month. '", "'. $ year. '"');
which is a stored procedure in my SQL, is it possible?
I'm trying to use the type of datatables that are here:
https://livewire-datatables.com/actions

**Comment by joshhanley** [comment]
@sixtyrc I think you'd be better off submitting this issue on the datatables package repo 🙂 https://github.com/mediconesystems/livewire-datatables
